### ISSUE TITLE
RC4 support

### ISSUE THREAD

**Issue by jdarling** [hapijs/wreck#115]
I know that Node v4+ disables RC4 support, but is there any way that we can enable RC4 (and other "depricated" transport ciphers) when making a Wreck request?  We have a lot of Java API's we call out to that are not yet upgraded to the latest and greatest standards.  Some may never upgrade :(

**Comment by jdarling** [comment]
Basically allowing pass-through of ciphers to tls.connect (https://nodejs.org/dist/latest-v4.x/docs/api/tls.html#tls_tls_connect_options_callback) would probably be enough.

**Comment by hueniverse** [comment]
Doesn't sound like a good idea.

**Comment by geek** [comment]
@jdarling we do allow the `secureProtocol` option, which is passed to OpenSSL.  Here is the documentation on the methods: https://www.openssl.org/docs/manmaster/ssl/ssl.html#DEALING_WITH_PROTOCOL_METHODS

Therefore, we do support any SSL method that your installation of OpenSSL supports.  Beyond that, you may be better off using the node https agent directly.

**Comment by jdarling** [comment]
Yeah, I was afraid that might be the answer :D.  I'm working with our teams to see if I can get the proxies in front of these services upgraded to the latest and greatest SSL/TLS, as a last resort I'll go to using https directly.  My hopes were maybe to get a pass through connectionOptions type of thing that would pass directly to the underlying HTTPS/HTTP connection.

**Comment by geek** [comment]
@jdarling did you have any luck working around the issue?

**Comment by jdarling** [comment]
Well, while I did figure out how I could hack the code to make it work, I got lucky and convinced our Security team that this was a problem that they should look at.  Then Chrome updated and basically blocked all non-proxied requests to the same services and lots of users broke (we proxy all our requests so our application didn't break).  At that point everyone updated to "modern" standards, and this became a non-issue.

While I understand the arguments against allowing direct access, I would still say that the option to pass down configuration values is a good idea.
